lib/data/dynamo/table.ts

```diff
@@ -118,15 +118,15 @@ export abstract class DynamoTable<TModel extends Model> implements ModelFactory<
         return new Promise(promiseCallback);
     }
 
-    public update(data: TModel, options: UpdateItemOptions = {}): Promise<TModel> {
+    public update(data: TModel, options: UpdateItemOptions = {}): Promise<boolean> {
         let promiseCallback = (resolve, reject) => {
 
             let callback: DynogelsItemCallback = (err, document) => {
                 if (err) {
                     console.error('Error updating item on ' + this.tableName + ' table. Err: ', err);
                     reject(err);
                 } else {
-                    resolve(data);
+                    resolve(true);
```

Comment:
Is this even necessary? I think we could just do an empty resolve if it succeeds and reject if not

Reply:
Just doesn't seem intuitive if you are calling this for the first time and have no idea what the boolean values returned mean. And also the fact that we would never return false